### PR TITLE
`function-name-case` rule

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: docs/user-guide/rules.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`function-comma-space-before`](../../src/rules/function-comma-space-before/README.md): Require a single space or disallow whitespace before the commas of functions.
 - [`function-linear-gradient-no-nonstandard-direction`](../../src/rules/function-linear-gradient-no-nonstandard-direction/README.md): Disallow direction values in `linear-gradient()` calls that are not valid according to the [standard syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient#Syntax).
 - [`function-max-empty-lines`](../../src/rules/function-max-empty-lines/README.md): Limit the number of adjacent empty lines within functions.
+- [`function-name-case`](../../src/rules/function-name-case/README.md): Specify lowercase or uppercase for functions.
```
Comment (jeddy3) [inline docs/user-guide/rules.md]: "for function names"

File: src/rules/function-name-case/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# function-name-case
+
+Specify lowercase or uppercase for functions.
```
Comment (jeddy3) [inline src/rules/function-name-case/README.md]: "for function names"

File: src/rules/function-name-case/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+```css
+      a { width: calc(5% - 10em); }
```
Comment (jeddy3) [inline src/rules/function-name-case/README.md]: Can we have this flush to the left please? We only align the arrow with the final letter in the comment when the comment is shorter than the code snippet.

**Comment by jeddy3** [comment]
Looks great to me! I've added a couple of very minor documentation amends.

**Comment by davidtheclark** [comment]
I see you have a system down.

**Pull request merged**